src/App.js

```diff
@@ -79,9 +71,11 @@ class App extends Component {
     return (
       <main>
         <NavBar />
+        <Routes>
+          <Route path="/" element={<Movies movies={this.state.movies} displayDetails={this.displayDetails}/>}/>
```

Comment:
The name of the prop `displayDetails` should be changed since it's only getting the details now. So we should also change the method name to something like `getDetails`.